README.md

```diff
@@ -1 +1 @@
-# Dell-projects
\ No newline at end of file
+Multithreading introductory task in Dell mentoring program.
```

Comment:
Please, add the purpose of the program (what it does) and usage (how to run it).